website/components/Theme.tsx

```
@@ -40,17 +40,23 @@
+              type="text/css"
+              dangerouslySetInnerHTML={{ __html: styleSheet }}
+            />
+          )
+          : null
+      ))}
+      {html && (
         <style
           type="text/css"
-          dangerouslySetInnerHTML={{ __html: styleSheet }}
+          id={`__DESIGN_SYSTEM_VARS-${id}`}
+          dangerouslySetInnerHTML={{ __html: html }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)